Assets/Scripts/Copilots/Copilots.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Copilots : MonoBehaviour
```

Comment:
Either comment on why this is a singleton class or rename the script to be more clear